spec/grape-active_model_serializers/formatter_spec.rb

```diff
@@ -65,5 +84,9 @@ def endpoint.default_serializer_options
       expect(subject.instance_variable_get('@only')).to eq([:only])
       expect(subject.instance_variable_get('@except')).to eq([:except])
     end
+
+    it 'should read serializer options like "root"' do
+      expect(described_class.build_options_from_endpoint(endpoint).keys).to include :root
```

Comment:
i don't really understand the spec here.  so, root is set to false and yet it should include `root` in the build options?